### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by poddus** [pdfarranger/pdfarranger#826] [General]
Hey there,

I use pdfarranger quite often and would like to send a thank you to the devs. you got a paypal or something? barring that, would you recommend any other project that you think deserves a donation?

cheers,
Leopold

**Comment by dreua** [comment]
Hey, your "thank you" has been received, thank you for your kind words. I'm really happy to hear that our software is useful to you!
For donations I don't think we as a community project accept those right now. It would be be some organizational overhead to decide the splitting or spending of those.

From the top of my head, the eff and fsf would be the well known organizations to send donations to in the general context of open source. Maybe also Mozilla or your favorite Linux distribution.

For the projects we build upon: pikepdf has a link to its open collective on the Github page. Qpdf would be a worthy receiver but not sure @jberkenbilt is accepting donations. There are of course many more libraries and tools we use to build PDF Arranger. Haven't thought of it at first, but the GNOME project is of course accepting donations: https://www.gnome.org/donate/

I'm looking forward to what the other regular contributors have to say about this, those are just my ideas regarding this topic.

**Comment by jberkenbilt** [comment]
Hello! People have occasionally made unsolicited donations for qpdf, but I generally discourage it as there are other causes that need it more. I would agree with @dreua 's list. I support both [EFF](https://eff.org) and [FSF](https://fsf.org) myself and have also donated to the Python Software Foundation and many others. I try to donate to an open source project every year. Recently I donated to nannou.cc via open collective since I used that for a personal project. I've also donated to KDE. Speaking for myself only, I have spent my career as a software developer and have benefited greatly from open source software. I give qpdf as a gift to the community as a way of giving something back without respecting anything in return, though expressions of gratitude are of course appreciated. :-)

**Comment by poddus** [comment]
thanks for your replies! I already donated to mozilla this year but after reading some articles from open source devs I thought I would try to show support for some of the smaller projects I use, since you guys really do make a big difference in the community but often go underappreciated. I understand it can be difficult working on a project without funding from the big players in the OSS ecosystem and I think we need more than just gnome and KDE for a vibrant linux desktop experience (though that is not meant in anyway to dismiss the great work they do)! In that light I think I'll be heading to pikepdf as the backend for pdfarranger, and for what it's worth: thank you very much for all you do!

**Comment by Justitia-Pax** [comment]
I am not a developer or anything like that. I just downloaded ElevenClock and it is perfect to create seconds that I can't on windows 11'own clock. While I was downloading and setting up - there was an opportunity to donate. I was more focussed on setting up & planned to come back to donate afterwards. But I am not able to find it again or anywhere else on your website. 
It's not a huge amount i want to donate -- but i still would like to - through PayPal.. 
I already donate to other free items: Mozilla, AdBlockPlus, etc. Just would like to the developer here to the list.

**Reply by dreua** [reply]
I don't see how this is relevant here. The product you are mentioning is not well known and this is a discussion about PDF arranger.
